### PR TITLE
[tests-only][full-ci] move files in `apiFullTextSearch` suites and renamed suites as `apiSearch`

### PR DESCRIPTION
**Pull request by nabim777**
## Description
In this PR, suite `apiFullTextsearch` renamed as `apiSearch` and search-related feature files are moved to single suit `apiSearch`.
The feature files which are moved and renamed are:
|Before    	|   After	|
|---	|---	|
|   `apiSpaces/search.feature` 	|   `apiSearch/apiSpaceSearch.feature`	|
|   `apiFullTextSearch/search.feature`	|  `apiSearch/contentSearch.feature` | 
|   `apiGraph/fullSearch.feature` 	|   `apiSearch/tagSearch.feature`	| 

## Related Issues:
https://github.com/owncloud/ocis/issues/7135

## How Has This Been Tested?
- test environment:
- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by SwikritiT — COMMENTED** [review]
LGTM

File: tests/acceptance/config/behat.yml [hunk 1]
```diff
@@ -119,7 +119,6 @@ default:
         - TUSContext:
         - SpacesTUSContext:
         - OcisConfigContext:
-        - TagContext:
```
Comment (saw-jan) [inline tests/acceptance/config/behat.yml:122]: Please, check if we can remove `SearchContext` from `apiSpaces` and `apiGraph`

Comment (nabim777) [reply tests/acceptance/config/behat.yml:122]: not needed so removed `SearchContext`

**Review by saw-jan — APPROVED** [review]

**Review by saw-jan — CHANGES_REQUESTED** [review]
Or better generalize the suite name to `apiSearch`
https://github.com/owncloud/ocis/blob/f3c0d26144e7e17c243db3ec4f2b284e8b41004b/tests/acceptance/config/behat.yml#L194-L196

**Review by saw-jan — APPROVED** [review]

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/25985/52/5
```
runsh: Total unexpected failed scenarios throughout the test run:
apiSpacesShares/copySpaces.feature:395
```
I will restart CI to see if that scenario can pass.

**Review by phil-davis — APPROVED** [review]

**Comment by nabim777** [comment]
- [ ] backport to the stable-3.0

**Pull request merged**